diff --git a/node_modules/@wagmi/connectors/dist/esm/metaMask.js b/node_modules/@wagmi/connectors/dist/esm/metaMask.js
index 53ef169..b490a7d 100644
--- a/node_modules/@wagmi/connectors/dist/esm/metaMask.js
+++ b/node_modules/@wagmi/connectors/dist/esm/metaMask.js
@@ -1,7 +1,7 @@
 import {} from '@metamask/sdk';
 import { ChainNotConfiguredError, createConnector } from '@wagmi/core';
 import { ResourceUnavailableRpcError, RpcError, SwitchChainError, UserRejectedRequestError, getAddress, numberToHex, } from 'viem';
-const isMobileBrowser = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
+const isMobileBrowser = typeof navigator != 'undefined' ? /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) : false;
 metaMask.type = 'metaMask';
 export function metaMask(parameters = {}) {
     let sdk;
